### PR TITLE
react-leaflet-draw updated to v3 react-leaflet 3.0.0

### PR DESCRIPTION
**Pull request by DVGY**
Please merge my branch

### CODE REVIEW

**Review by agusterodin — COMMENTED** [review]
Two things I notice:

Stray console.log left behind and i don't think the library author will be happy that the quotes all got changed from single to double. Run eslint fix to take care of this so that it conforms to the linter setup he has defined.

**Comment by agusterodin** [comment]
To be clear: not just the double quotes. I am guessing you ran this through a tool like prettier. It creates a lot of 'noise' in the changes. If you do use a tool like prettier be sure to also eslint fix. Doing so will greatly increase the chance that the repo owner will accept your PR.

Thanks for your hard work. Much appreciated!

File: example/edit-control.js [hunk 1]
```diff
@@ -1,129 +1,132 @@
...
-              onEditStop={this._onEditStop}
-              onDeleteStart={this._onDeleteStart}
-              onDeleteStop={this._onDeleteStop}
-              draw={{
-                rectangle: false
-              }}
-            />
+        <FeatureGroup
+          ref={(reactFGref) => {
+            console.log(reactFGref);
```
Comment (srghma) [inline example/edit-control.js:88]: Could you please remove

File: example/edit-control.js [hunk 2]
```diff
@@ -1,129 +1,132 @@
...
+  _editableFG = null;
 
   _onFeatureGroupReady = (reactFGref) => {
-
     // populate the leaflet FeatureGroup with the geoJson layers
 
     let leafletGeoJSON = new L.GeoJSON(getGeoJson());
-    let leafletFG = reactFGref.leafletElement;
+    let leafletFG = reactFGref;
+    console.log({ reactFGref });
```
Comment (srghma) [inline example/edit-control.js:119]: Another one

File: src/EditControl.js [hunk 3]
```diff
@@ -1,141 +1,107 @@
...
-  }
-
-  onDrawCreate = (e) => {
-    const { onCreated } = this.props;
-    const { layerContainer } = this.props.leaflet;
+  drawRef.current = createDrawElement(props, context);
 
-    layerContainer.addLayer(e.layer);
+  const onDrawCreate = (e) => {
+    console.log(e.layer);
```
Comment (srghma) [inline src/EditControl.js:32]: Another one

File: src/EditControl.js [hunk 4]
```diff
@@ -1,141 +1,107 @@
...
           let handler = handlers[0];
-          this.props[handler] && this.props[handler](evt);
+          props[handler] && props[handler](evt);
         }
       });
     }
+    map.on(leaflet.Draw.Event.CREATED, onDrawCreate);
 
-    map.on(leaflet.Draw.Event.CREATED, this.onDrawCreate);
+    console.log({ drawRef });
```
Comment (srghma) [inline src/EditControl.js:57]: Another one log

**Review by srghma — APPROVED** [review]
Could you please remove extra logs or add useful message as it is done with already existing logs

**Comment by DVGY** [comment]
> Could you please remove extra logs or add useful message as it is done with already existing logs

Yes, please I will take care of this. Give me some time and I will respond with all the new changes

File: example/edit-control.js (hunk 1 above)
Comment (DVGY) [reply example/edit-control.js]: I changed this

File: example/edit-control.js (hunk 2 above)
Comment (DVGY) [reply example/edit-control.js:117]: Removed console.log

File: src/EditControl.js (hunk 3 above)
Comment (DVGY) [reply src/EditControl.js]: Removed console.log

File: src/EditControl.js (hunk 4 above)
Comment (DVGY) [reply src/EditControl.js]: Resolved

**Comment by DVGY** [comment]
> To be clear: not just the double quotes. I am guessing you ran this through a tool like prettier. It creates a lot of 'noise' in the changes. If you do use a tool like prettier be sure to also eslint fix. Doing so will greatly increase the chance that the repo owner will accept your PR.
> 
> Thanks for your hard work. Much appreciated!

I removed all the double quotes

**Pull request merged**